tools/springboard-alerts/.rubocop.yml

```diff
@@ -0,0 +1,10 @@
+Layout/IndentationWidth:
+  Width: 4
+Layout/IndentFirstHashElement:
+  Enabled: false
+Metrics/LineLength:
+  Enabled: false
+Style/FrozenStringLiteralComment:
+  Enabled: false
+Layout/EndOfLine:
+  EnforcedStyle: lf
```

Comment:
I don't really like linters and would prefer that we don't have one.  

If it helps your team, let's keep this.

Definitely change the indentation with to 2.

Reply:
I prefer to keep style-linter if you don't mind :)
Fixed indentation to 2.